crates/context_aware_config/src/api/context/helpers.rs

```diff
@@ -109,7 +109,7 @@ fn get_functions_map(
 }
 
 pub fn validate_value_with_function(
-    fun_name: &str,
+    _fun_name: &str,
```

Comment:
If the name is not used, we could just remove it? This is beyond the scope of the PR, but a suggestion. If the warning isn't present, will we remember to remove this?